### ISSUE TITLE
(Feature request) Add (X) button to close Sidebar views somewhere

### ISSUE THREAD

**Issue by Kadigan** [lhc70000/iina#1274]
I can open these panels just fine, but they then linger on the screen w/ no obvious way to close them.

**System and IINA version:**

- macOS 10.12.6
- IINA 0.0.14 build 47
- mpv 0.27.0 (Homebrew)

**Rationale:**

Once a Sidebar is open, you can either de-focus and re-focus on the window w/o moving the mouse, and then it _should_ close by itself in a second... or use a keyboard shortcut, depending on the Sidebar view currently open. I don't use these keyboard shortcuts, because there are UI buttons for everything.

Except a (X) button on the Sidebar, that is.

A clearly visible (X) button somewhere on the Sidebar would clearly communicate how to close the Sidebar views.

**Comment by lhc70000** [comment]
You can close the side panel by simply clicking anywhere outside it.

**Comment by Kadigan** [comment]
Unfortunately, no. It takes several clicks for this to happen, and sometimes the damn Sidebar persists until I focus away from the application and back again.

There seems to be no clear correlation (for me) between clicking outside the Sidebar and the Sidebar closing. The number of clicks, the delay, and the cursor position - are all inconsistent between attempts.

**Comment by lhc70000** [comment]
@Kadigan Are you using a Magic Mouse?

**Comment by Kadigan** [comment]
Yes. This is a MacPro5,1 with a Magic Mouse, secondary click enabled.

**Comment by lhc70000** [comment]
@Kadigan So that should be the problem in #39 and #96. #307 fixed this issue, and I'm considering merging it recently.

**Comment by Kadigan** [comment]
I would like to note that I can use full-screen _perfectly fine_ using double-click (works every time); there are no similar issues in other software for me either. I do not see the cursor move _at all_, when I click...

**However**, I have just tested this with a _cable_ mouse (no-name generic one I had around for emergency purposes), and _the problem goes away_. So it _is clearly related_ to the Magic Mouse after all, it seems.

**Comment by Kadigan** [comment]
I would like to point out, however, that this would have never escalated into a reportable issue had there been other _obvious_ ways to close the Sidebar (like an (X) button). Clicking on the display sure is convenient, but when it fails, the backup (keyboard shortcuts) is not obvious and requires awareness of which panel you currently have open (there are different shortcuts for different panels, and while using the same twice is a workable work-around, it's not elegant).

So while having the left-click fixed on Magic Mice would be good, I stand by my original assessment - please add an obvious UI element to close the panels.

**Comment by uiryuu** [comment]
Fixed in #6132.

**Comment by uiryuu** [comment]
This feature has been partially implemented in the `develop` branch and will be included in the next major release of IINA (v1.5.0). Further improvements, adjustments, and bug fixes related to this issue are expected. If you're interested, you can try it out by downloading a [Nightly build](https://iina.io/nightly/) (a GitHub account is required).
